x-pack/plugins/security_solution/public/timelines/components/timeline/body/renderers/reason_column_renderer.tsx

```diff
@@ -82,6 +83,7 @@ const ReasonCell: React.FC<{
 
   const rowRender = useMemo(() => {
     return (
+      timelineId !== TimelineId.rulePreview &&
```

Comment:
Putting the logic in the row renderer itself was just a faster way to make the boolean logic work, I could definitely break it out into its own thing. It's mainly triggering [this logic](https://github.com/elastic/kibana/pull/138312/files#diff-9ef03aaad570456e3a1596cc1de6d7c2f343f247c9715e89022a41ed180386a1R98) to return false and not enrich the `ReasonCell` component. Since we currently don't want any cells in the rule preview table to be enriched this seemed like a good approach for a quick fix although I agree that later on if we want it to be a bit more granular, having a `readOnly` prop or something would be a good idea

Reply:
> having a `readOnly` prop or something would be a good idea

@dplumlee ++ I was thinking about the same thing. Rule Preview component could pass via props "hey, render it with row renderers, but the rendered rows should be read-only, i.e. don't have active click handlers etc".

Would you be comfortable with opening some tickets for that? I guess one would be for tech debt/improvement of the renderer: it shouldn't know about its parent components. The other one would be for implementing a read-only renderer for the Reason column.